api/src/objects/drop.rs

```diff
@@ -124,16 +121,6 @@ impl Drop {
             },
         }
     }
-
-    /// A list of all NFT purchases from this drop.
-    async fn purchases(&self, ctx: &Context<'_>) -> Result<Option<Vec<purchases::Model>>> {
-        let AppContext {
-            drop_purchases_loader,
-            ..
-        } = ctx.data::<AppContext>()?;
-
-        drop_purchases_loader.load_one(self.drop.id).await
-    }
```

Comment:
can we keep this but have it return mintHistories? mark the field as deprecated.

Reply:
shouldn't the purchases field returns Purchase object?